app/controllers/carts_controller.rb

```diff
@@ -1,19 +1,16 @@
 class CartsController < ApplicationController
-  before_action :set_category, only: %i[show update destroy]
+  load_and_authorize_resource
+  before_action :set_category, only: %i[update destroy]
 
   def create
     Cart.create(cart_params)
   end
 
-  # display all elements of model
+  # users/1/carts
   def index
     @cart = Cart.all
```

Comment:
Please, avoid using 'all' method without any limits for now. Use @user = User.all.limit(10) for example or you can load more than 10. For the future look at one of these gems for pagination https://github.com/ddnexus/pagy or https://github.com/kaminari/kaminari